torch/_inductor/cudagraph_utils.py

```diff
@@ -234,3 +240,12 @@ def log_data_ptr_mismatch(
                 f"input stack trace: {get_placeholder_stack_trace(placeholder)}\n"
             )
     return error_msg
+
+
+def maybe_get_static_data_ptr(
+    idx: int, inputs: List[Union[torch.Tensor, int]], static_input_idxs: List[int]
+) -> Optional[int]:
+    inp = inputs[idx]
```

Comment:
nit.. would maybe keep this a little closer to the source, still in cudagraph_trees